notebooks/MAST/astroqueryMAST/beginner_search_astroqueryMAST/beginner_search_astroqueryMAST.ipynb

```diff
@@ -0,0 +1,395 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This will give a warning that the coordinates are being interpreted as an ICRS coordinate provided in degrees\n",
+    "obsByRegion = Observations.query_region(\"322.49324 12.16683\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# How many results? Let's look at the first ten.\n",
+    "print(\"Number of results:\",len(obsByRegion))\n",
+    "print(obsByRegion[:10])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2. By Object Name\n",
+    "\n",
+    "The `query.object` method allows you to search based on object name and optional radius argument.  \n",
+    "The object name is first resolved to sky coordinates, and then the resolved region is queried as above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "obsByName = Observations.query_object(\"M51\",radius=\".005 deg\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Number of results:\",len(obsByName))\n",
+    "print(obsByName[:10])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. By Other Criteria (with or without name/region) <a id=\"crit\"></a>\n",
+    "\n",
+    "To search for observations based on additonal parameters, you can use `query_criteria`. In a sense, this is a more powerful version of the tools above, as you can still search by coordinates and objectname; however, you can inculde additional desired criteria. You may also run the search without specifying a name or coordinates.\n",
+    "\n",
+    "To perform the search, give your critera as keyword arguments. Valid criteria are listed [here](https://mast.stsci.edu/api/v0/_c_a_o_mfields.html). Some examples are: \"coordinates\", \"t_exptime\" (exposure time), and \"s_dec\" (source declination)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "obsByCriteria = Observations.query_criteria(obs_collection=[\"HLA\"], s_dec=[50,60], \n",
+    "                                            calib_level=[3], proposal_pi=\"Mould*\", \n",
+    "                                            dataproduct_type=\"IMAGE\", t_max=[49800,49820])\n",
+    "\n",
+    "#obsByCriteria = Observations.query_criteria(obs_id = 'hst_05766_04_wfpc2_total_wf')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "obsByCriteria.show_in_notebook(display_length=5)"
```

Comment:
Assuming you split the above, I would put this in-between (e.g., after the query_criteria() and before you show how to use query_criteria for a single obs_id.  The first query_criteria() ony returns 4 objects, so setting display_length=5 is unnecessary.  If we want to highlight this option, I would modify the query to return a couple dozen results so setting the number per page in the table is helpful.  Maybe show how to search for more than one obs_collection?  HLA + HST + one other?